roles/openshift_registry/tasks/main.yml

```diff
@@ -1,11 +1,14 @@
 ---
-- set_fact: _oreg_images="--images={{ oreg_url|quote }}"
-  when: oreg_url is defined
+
+- set_fact: _oreg_images="--images='{{ openshift.master.registry_url }}'"
+
+- set_fact: _oreg_selector="--selector='{{ openshift.master.registry_selector }}'"
 
 - name: Deploy OpenShift Registry
   command: >
     {{ openshift.common.admin_binary }} registry
-    --create
-    --credentials={{ openshift_master_config_dir }}/openshift-registry.kubeconfig {{ _oreg_images|default() }}
+    --create --service-account=registry
+    {{ _oreg_selector }} --replicas=0
```

Comment:
I'm not sure about this one yet. I have a second playbook for Online that sets up our backend storage with S3 before settings replicas=1. (Located here: https://github.com/openshift/openshift-ansible/blob/master/playbooks/adhoc/s3_registry/s3_registry.yml )

I'll do a test to see what happens to the registry when we scale up before backend storage is in place, to ensure this will work for Online. I'm guessing I can just scale it back down to 0 in my S3 playbook, then set up backend storage, then scale it back up.